riocli/project/inspect.py

```diff
@@ -39,7 +39,29 @@ def inspect_project(format_type: str, project_name: str,
     try:
         client = new_v2_client(with_project=False)
         project = client.get_project(project_guid)
-        inspect_with_format(unmunchify(project), format_type)
+        inspect_with_format(to_manifest(unmunchify(project)), format_type)
     except Exception as e:
         click.secho(str(e), fg=Colors.RED)
         raise SystemExit(1)
+
+def to_manifest(project: typing.Dict) -> typing.Dict:
+    """
+    Transform a project resource to a rio apply manifest construct
+    """
+    return {
```

Comment:
I think we may not need the `to_manifest` as API's output is directly in this format.